app/src/main/java/net/programmierecke/radiodroid2/PlayerService.java

```diff
@@ -408,6 +418,7 @@ public int onStartCommand(Intent intent, int flags, int startId) {
             MediaButtonReceiver.handleIntent(mediaSession, intent);
         }
 
+        enableMediaSession();
```

Comment:
Why do you want to unconditionally enable it here? It's definitely wrong.

Reply:
This was necessary to make also PLAY work consistently and not only after PAUSE. Something like this, we need, I guess. Media keys are also only sent to MediaSessionCompat if mediaSession is set to active.

Reply:
I found a documentation for what you trying to do https://developer.android.com/guide/topics/media-apps/mediabuttons - you can react on media keys when mediasession is inactive.

Reply:
Thanks! I will try it, don't know when, however.

Reply:
You can add it in other PR and make this PR only about skip to prev/next.

Reply:
Yes, good idea. I've done it like that now and have hopefully fixed all issues.
(I have however left the code to start with the previous station from the history if nothing was played before in the current app instance).

Reply:
Thanks again for the hint. It has worked out of the box.